babel.config.js

```diff
@@ -1,5 +1,3 @@
 module.exports = {
-  presets: [
-    '@vue/app'
-  ]
-}
+  presets: [ [ "@vue/app", { useBuiltIns: "entry" } ] ]
```

Comment:
I am confused as to why these old things are showing up as changes to the 'master' branch, as this is change is already on the 'master' branch. Makes me worried that something catastrophic is about to happen.

Reply:
Hmm - I thought I had synced my forked branch with the master but there are way too many conflicts and changes showing up.  I may delete this request, re-sync, and resubmit.  The only conflicts should be in Readme.md and src/assetts/mapStyles/mapStyles.js.  Everything else should just be additional docker/jenkins files plus config.sh.   I may not get to it before Friday or the weekend depending on my system upgrade/rebuild today.